variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "prefix" {
+    description = "The prefix used for all resources in this environment"
+}
+
+variable "location" {
+    description = "The Azure location where all resources in this deployment should be created"
+    default = "uksouth"
+}
+
+variable "flask_app" {}
+variable "docker_enable_ci" {}
+variable "docker_registry_server_password" {}
```

Comment:
You can also mark variables as sensitive to avoid them accidentally getting logged or included in a non-sensitive output, though currently there is no problem.